internal/domain/product.go

```diff
@@ -0,0 +1,60 @@
+package domain
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+var ProductCategory = []string{"Clothing", "Accescories", "Footware", "Beverages"}
```

Comment:
di buatin variable 1 1 mar. supaya bisa direuse

```suggestion
var (
  ProductCategoryClothing = "Clothing"
)

var ProductCategories = []string{
   ProductCategoryClothing
}
```

Reply:
done